geonode_mapstore_client/client/js/routes/Redirect.jsx

```diff
@@ -85,7 +67,14 @@ function CatalogueRoute({
     });
 
     const parsedPlugins = useMemo(() => ({ ...loadedPlugins, ...getPlugins(plugins) }), [loadedPlugins]);
-    const className = `gn-catalogue-page`;
+    const className = `gn-redirect-page`;
+
+    const search = location?.search ?? {};
+    const catalogHomeRedirectsTo = getGeoNodeLocalConfig('geoNodeSettings.catalogHomeRedirectsTo');
+    if (!isEmpty(catalogHomeRedirectsTo)) {
+        window.location.href = `${catalogHomeRedirectsTo}#/${search ? search : ""}`;
+        return null;
+    }
 
     return (
```

Comment:
Do we need a return for this component?  (probably this need just to return null)
Please remove all not needed props and component

Reply:
only null return